src/pdf.rs

```diff
@@ -50,11 +50,11 @@ impl PdfSurface {
 
     pub fn get_versions() -> impl Iterator<Item = PdfVersion> {
         let vers_slice = unsafe {
-            let mut vers_ptr: *mut ffi::cairo_pdf_version_t = mem::uninitialized();
+            let mut vers_ptr = mem::MaybeUninit::<*mut ffi::cairo_pdf_version_t>::uninit();
```

Comment:
`<*mut ffi::cairo_pdf_version_t>::` seems unneeded here and in other places

Reply:
True but explicit doesn't hurt I guess? :)

Reply:
Agree

Reply:
This one here is a pointer (to a pointer of versions), and we use `ptr::null()` or `ptr::null_mut()` for unitialized pointers elsewhere so it might be better to do that for consistency here too.